src/components/Button/BaseButton.tsx

```diff
@@ -120,6 +123,11 @@ export const BaseButton: FC<InternalButtonProps> = React.forwardRef(
             { [styles.buttonText3]: size === ButtonSize.Small },
         ]);
 
+        const badgeClassNames: string = mergeClasses([
+            styles.counter,
+            buttonTextClassNames,
```

Comment:
Not sure why this is needed. Whats the reason for this ?

Reply:
The badge layout requires flex support and to inherit the font size from button.